docs/SettingUp.adoc

```diff
@@ -34,7 +34,6 @@ Do not disable them. If you have disabled them, go to `File` > `Settings` > `Plu
 . Locate the `build.gradle` file and select it. Click `OK`
 . Click `Open as Project`
 . Click `OK` to accept the default settings
-. Open a console and run the command `gradlew processResources` (Mac/Linux: `./gradlew processResources`). It should finish with the `BUILD SUCCESSFUL` message. +
 This will generate all resources required by the application and tests.
```

Comment:
This sentence also needs to be deleted as well.

Reply:
You missed this comment:
> This sentence also needs to be deleted as well.

This was added in 1488051e2795c1e41d82850f9, with obvious reference to the `processResources` task, so if the processResources task is being removed it makes sense to remove this as well.

Anyway, it is not precise to say that "pressing OK would generate all resources required by the application and tests." anyway, since we don't have any resources to generate.